### PR TITLE
Fix GGA: gnssMethodQuality is numeric

### PR DESCRIPTION
**Pull request by free-x**
Hi, 
here is fix for correct construction of GGA set with GPS quality as integer

Fixes #40 

Regards
free-x

### CODE REVIEW

File: sentences/GGA.js [hunk 1]
```diff
@@ -82,12 +82,42 @@ module.exports = function (app) {
         gnssDifferentialReference = ''
       }
 
+      switch (gnssMethodQuality) {
```
Comment (tkurki) [inline sentences/GGA.js]: ignssMethodQuality variable should be declared.

**Comment by free-x** [comment]
Hi @tkurki ,

I'm desperate. I've tried writing a test for GGA and it somehow doesn't work.
I don't know why something like that just delivers zero.
```
     app.streambundle.getSelfStream('navigation.gnss.methodQuality').push('DGNSS fix')
     app.streambundle.getSelfStream('navigation.gnss.satellites').push(6)
     app.streambundle.getSelfStream('navigation.gnss.horizontalDilution').push(1.2)
     app.streambundle.getSelfStream('navigation.gnss.antennaAltitude').push(18.893)
     app.streambundle.getSelfStream('navigation.gnss.geoidalSeparation').push(-25.669)
     app.streambundle.getSelfStream('navigation.gnss.differentialAge').push(2.0)
     app.streambundle.getSelfStream('navigation.gnss.differentialReference').push('0031')
```
But "navigation.datetime" and "navigation.position" seems to work

Result looks like this:
$IIGGA,172814,3723.4659,N,12202.2696,W,0,0,0,0,M,0,M,,*52

Any Ideas?
Regards
free-x

File: test/GGA.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     const onEmit = (event, value) => {
+        assert.equal(value ,'$IIGGA,172814,3723.4659,N,12202.2696,W,0,0,0,0,M,0,M,,*52')      
+        done()
+     }
+     const app = createAppWithPlugin(onEmit, 'GGA')
+     app.streambundle.getSelfStream('navigation.datetime').push('2015-12-05T17:28:14Z')
+     app.streambundle.getSelfStream('navigation.position').push({ longitude: -122.03782631066667, latitude: 37.39109795066667 })
+  })
+
+/*  it('works with sample values', done=> {
```
Comment (tkurki) [inline test/GGA.js]: What about this commented test?

Comment (free-x) [reply test/GGA.js]: I don't know what to do. I am at the end with my Latin. I don't succeed in feeding the test with values. That's why I commented this part out. 
```
      AssertionError [ERR_ASSERTION]: '$IIGGA,172814,3723.4659,N,12202.2696,W,0,0,0,0,M,0,M,,*52' == '$IIGGA,172814.0,3723.46587704,N,12202.26957864,W,2,6,1.2,18.893,M,-25.669,M,2.0,0031*58'
      + expected - actual

      -$IIGGA,172814,3723.4659,N,12202.2696,W,0,0,0,0,M,0,M,,*52
      +$IIGGA,172814.0,3723.46587704,N,12202.26957864,W,2,6,1.2,18.893,M,-25.669,M,2.0,0031*58
      
      at Object.onEmit [as emit] (test/GGA.js:19:16)
      at plugin.unsubscribes.push.Bacon.combineWith.filter.changes.debounceImmediate.onValue.nmeaString (index.js:54:17)
      at node_modules/baconjs/dist/Bacon.js:1048:16
      at node_modules/baconjs/dist/Bacon.js:463:23
      at processAfters (node_modules/baconjs/dist/Bacon.js:359:11)
      at Object.inTransaction (node_modules/baconjs/dist/Bacon.js:439:9)
      at Dispatcher.push (node_modules/baconjs/dist/Bacon.js:1214:24)
      at Dispatcher.handleEvent (node_modules/baconjs/dist/Bacon.js:1267:17)
      at Bus.sink (node_modules/baconjs/dist/Bacon.js:242:17)
      at Bus.push (node_modules/baconjs/dist/Bacon.js:2371:21)
      at Context.done (test/GGA.js:24:60)
```

Comment (tkurki) [reply test/GGA.js]: Push position last, after all the other values.

Comment (free-x) [reply test/GGA.js]: thx. that's it!!! 
Another small question: formating of output 
For Example: If I define
app.streambundle.getSelfStream('navigation.gnss.differentialAge').push(2.0)
Result ist integer 2.
How to keep float 2.0?

**Comment by free-x** [comment]
@tkurki 

I think, I'm done. If you want, you can merge now

Regards
free-x

**Pull request merged**